src/App.jsx

```diff
@@ -0,0 +1,15 @@
+import React, { useState } from 'react';
+
+import Page from './Page';
+
+export default function App() {
+  const [count, setCount] = useState(0);
+
+  function handleClick(i = 1) {
+    setCount(count + i);
+  }
+
+  return (
+    <Page count={count} onClick={(i) => handleClick(i)} />
```

Comment:
```suggestion
    <Page count={count} onClick={(handleClick} />
```
이렇게 사용하게 되면 더 간단하게 사용할수 있을거 같네요 :)